frontend/src/views/SubjectCourses.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="main-content subject-course">
+    <AppBreadCrumb/>
```

Comment:
Broadly I think this page looks really good well done.

On larger widths though you can easily use more column space. I'm at work on a 1600 which isn't the average by any means but, yeah I think a bit more space would be good. For this type of exacting layout grid is potneitlaly better as a layout op.

Reply:
I barely know what i'm doing tbh - css just makes me hate my life cos i cbf to learn it properly.

In general though I think the main-content class should be wider which will make this whole thang look better, and can have 3 things per column on these pages.

Reply:
<3. It's cool. It looks good. Just providing feedbackssss.